File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/ICD10EstimateFilter.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System;

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class ICD10EstimateFilter
    {
        public EventTypeEnum EventType { get; set; }
        public string Icd10Codes { get; set; }
        public DateTime ReportDate { get; set; }
        public int Icd10DiagnosticGroupId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/common/icd10-estimate-filter.ts
import { EventTypeEnum } from '../../../../claimcare/src/app/claim-manager/shared/enums/event-type-enum';

export class ICD10EstimateFilter {
    eventType: EventTypeEnum;
    icd10Codes: string;
    reportDate: Date | string;
    icd10DiagnosticGroupId: number;
}

